### DISCUSSION TITLE
5.1.0 release timelines

### DISCUSSION THREAD

**Discussion by Krishnaj20** [Sunbird-Ed/Community#260] [General Discussion]
All - 5.1.0 regression cycle which was supposed to start on 17th Nov will start on 24th Nov resulting in a delay of 6-7 days to the 5.1.0 release. This is primarily due to the additional time needed by the building blocks to complete their CSP related changes. Pls plan accordingly. 

@Sunbird-Ed/sunbirded_workinggroup

@rhwarrier @surendrasinghs @luckynvdu @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 
@vinukumar-vs @pallakartheekreddy @reshmi-nair @anandp504 @vijiurs @SanthoshVasabhaktula @vrayulu @ashokreddy1208 @gandham-santhosh @kameshbhr @mmanimegalai @chitranshu-keshav @maheshkumargangula @sharathkashyap @ShailandraOjha

**Comment by ShailandraOjha** [comment]
We are yet to start with the 5.1 regression in the Sunbird stage environment as the Sunbird knowlg is yet to provide the sign-off and is expected to be provided by today.

We will be starting with the regression tomorrow.

@rhwarrier @surendrasinghs @luckynvdu @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789
@vinukumar-vs @pallakartheekreddy @reshmi-nair @anandp504 @vijiurs @SanthoshVasabhaktula @vrayulu @ashokreddy1208 @gandham-santhosh @kameshbhr @mmanimegalai @chitranshu-keshav @maheshkumargangula @sharathkashyap @https://github.com/anunagaraj

**Comment by anunagaraj** [comment]
We have not yet started the R5.1.0 Regression as sign off is pending from Inquiry and Knwlg BB . Where we have 2 open issues from Inquiry and 4 open issues from Knwlg, we do not have ETA for these open issues. 

We will be starting regression once we get sign off from knwlg and inquiry BB.

@rhwarrier @surendrasinghs @luckynvdu @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789
@vinukumar-vs @pallakartheekreddy @reshmi-nair @anandp504 @vijiurs @SanthoshVasabhaktula @vrayulu @ashokreddy1208 @gandham-santhosh @kameshbhr @mmanimegalai @chitranshu-keshav @maheshkumargangula @sharathkashyap @ShailandraOjha
